portal/src/components/v2/WhiteButton/WhiteButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import cn from "classnames";
+import { Button } from "@radix-ui/themes";
+import styles from "./WhiteButton.module.css";
+
+export interface WhiteButtonProps {
+  size: "1" | "2" | "3" | "4";
+  disabled?: boolean;
+  loading?: boolean;
+  text?: React.ReactNode;
+
+  type?: "button" | "reset" | "submit";
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export function WhiteButton({
```

Comment:
This WhiteButton has hover state, but when it is pressed (i.e. active state), it is visually equal to normal state. I cross checked PrimaryButton, PrimaryButton does not have this issue.

Reply:
The normal color is accent/9, and active color is accent/10. So it is because the generated colors are similar if accent color is #ffffff. Maybe we can let designer to try it in the storybook and see if any feedback.

Reply:
Actually in the Figma file, there is a pressed state in PrimaryButton. But WhiteButton does not have it. I think that's a problem, right?

Reply:
Yes. I think I will remind designer to take a look. But I believe finally the active color will follow radix button active color, which is accent/10.

Reply:
I've checked the actual implementation and the radix design file, and I think they are not 100% match.
The actual implementation have a `background-color: var(--accent-10)`, with a `filter: brightness(1.08)`, while the figma use a opacity to do it.
I think as long as we are clear about the button is implemented with radix button, and all behaviors should follow the button of radix button, it will be enough.